assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -183,32 +230,33 @@ function Carousel() {
             const isCurrentPage = index === currentPageIndex;
 
             return (
-              <DraggablePage
-                key={index}
-                onClick={handleClickPage(page)}
-                ariaLabel={
-                  isCurrentPage
-                    ? sprintf(
-                        __('Page %s (current page)', 'web-stories'),
-                        index + 1
-                      )
-                    : sprintf(__('Go to page %s', 'web-stories'), index + 1)
-                }
-                isActive={isCurrentPage}
-                pageIndex={index}
-                ref={(el) => {
-                  pageRefs.current[page.id] = el;
-                }}
-                width={PAGE_THUMB_WIDTH}
-                height={PAGE_THUMB_HEIGHT}
-              />
+              <Li key={index} marginRight={itemMarginRight}>
```

Comment:
This breaks semantics now [as _Each child of a `listbox` should have a role of `option`_](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role).  But the direct child of the `listbox` is not  the interactive element - that's a child further down the stack. I don't know what's best practice here though.

If at all possible, see if you can untangle by making the page indicator (whether draggable page preview or small compact indicator) the direct child of the list. If not, we might have to re-think the semantics for this one.

Reply:
Will be handled in https://github.com/google/web-stories-wp/issues/10